### ISSUE TITLE
Different frame numbers between annotations and images

### ISSUE THREAD

**Issue by lyliutwo** [wangxiao5791509/TNL2K_evaluation_toolkit#8]
I find some cases that have different frame numbers between the bbox annotations and images. 
cases in training set:
**{file_name: frames in annotation, frames of images}:**
{'Assassin_video_5-Done': [1080, 785],
 'BF5_FireGUN_video_01_done': [1266, 633],
 'Baseball_game_002-Done': [714, 713],
 'Boat_video1-Done': [570, 470],
 'Cat_BBC_video_03-Done': [145, 133],
 'catNight_video_B04_done': [1518, 151],
 'Fight_video_02-Done': [916, 914],
 'Fight_video_04-Done': [817, 816],
 'Joker_video_X02_done': [2139, 1268],
 'maliao_4-Done': [1609, 1608],
 'monitor_bikeyellow': [930, 929],
 'videoPlayer_video_01_done': [2009, 2008]}

I haven't check the test set.

**Comment by wangxiao5791509** [comment]
@lyliutwo 
Hi, thanks for your information. Please check this note in the **README**: 
      The annotations of 12 videos in the training subset are modified for more accurate annotation. Please update these videos with the [[new annotations](https://github.com/wangxiao5791509/TNL2K_evaluation_toolkit/tree/main/annos/revised_annotations.zip)].  


If you also find issues in the testing subset, please let us know. Thanks for your attention.

**Comment by lyliutwo** [comment]
Thank you very much! I will update the annotations
